docker-compose.deploy.yml

```diff
@@ -0,0 +1,61 @@
+version: '3.7'
```

Comment:
Version 3.7 is used here, but in other files, we use 3.8. 
Please explain why

Reply:
I know this was intended, but can't exactly remember why, I'll try upgrading it to 3.8 later today to see why this is 3.7, or in case it doesn't matter change it to 3.8

Reply:
This is because the pipeline does not install Docker from it's official repositories, so it does not have the latest version. When there are breaking changes, we will install the latest Docker version in our pipeline. But for now it doesn't have any benefits, it only increases the time of the pipeline.